### PR TITLE
Oryx-Stack-Updates | 20240912.2

### PR DESCRIPTION
**Pull request by Saipriya-1144**
Updated ASPNET_CORE_APP_60 from 6.0.32 to 6.0.33
Updated ASPNET_CORE_APP_80 from 8.0.7 to 8.0.8
Updated ASPNET_CORE_APP_90 from 9.0.0-preview.6.24328.4 to 9.0.0-rc.1.24452.1
Updated NET_CORE_APP_60 from 6.0.32 to 6.0.33
Updated NET_CORE_APP_80 from 8.0.7 to 8.0.8
Updated NET_CORE_APP_90 from 9.0.0-preview.6.24327.7 to 9.0.0-rc.1.24431.7
Updated node20Version from 20.15.1 to 20.17.0
Updated php82Version from 8.2.21 to 8.2.23
Updated php83Version from 8.3.9 to 8.3.11
Updated python310Version from 3.10.14 to 3.10.15
Updated python311Version from 3.11.8 to 3.11.10
Updated python312Version from 3.12.2 to 3.12.6
Updated python38Version from 3.8.19 to 3.8.20
Updated python39Version from 3.9.19 to 3.9.20

### CODE REVIEW

File: platforms/dotnet/versions/bookworm/versionsToBuild.txt [hunk 1]
```diff
@@ -21,8 +21,13 @@
...
 8.0.100-rc.2.23502.2, 45f09e7b031f4cf5b4dcead240fe47e2e3731d97d22aa96d3a02a087322658606cc22792053c3784c44f15d7c9bad0ac9dbda90def7b4e197f2955dca9a5bb6c,
 8.0.101, 26df0151a3a59c4403b52ba0f0df61eaa904110d897be604f19dcaa27d50860c82296733329cb4a3cf20a2c2e518e8f5d5f36dfb7931bf714a45e46b11487c9a,
+8.0.108, 5666ddf6fa9b65deaba4d7c5fcc2e2d56f631c4f5f6fb2a9f5919af0616ab2b420b12a828becc2e4b8628a76ac3dae824b55abde5c6d5ac59ee131d7eceae7c2,
 8.0.201, 310cf54f595698435b533931b12f86d49f89d27243cf7c87a5b926e0c676b80e869aa58aaff17b5095536c432f377c67d92bf0ca8941b9d891d4b3879637d488,
 8.0.202, e0e790c7cc6f8129913317d326c599ff8e8ed4927d4e0adccbe55c50be5c353fe3d83043e529973ced2b302b8432c2ab31533b94ffe9c363eaa9964a7160643a,
 8.0.301, 6e2e1ad5fe3f00e6974ad3eac9c5b74cd09521f19e06eb9aff45a44d6c55e4a2c1cd489364735215d2ea53cec2a7d45892a5ede344a8421be9ad15872c3496a2,
 8.0.303, 814ff07ccdfc8160c4a24adfda6c815e7feace88c59722f827a5a27041719067538754911fc15cb46978e16566fe0938695891723d182055190e876131faedda,
+8.0.304, 971c344379240ec4bfaaf1eca69c6667e594cdd0dfdcde6e8962cb7a41d669dff91c644e48eed3573d841b7b3e60ce02e0c27a7ce37b66cdec27bf3457087c4a,
```
Comment (CodingIsBliss) [inline platforms/dotnet/versions/bookworm/versionsToBuild.txt:29]: lets only update the latest sdks

**Review by CodingIsBliss — DISMISSED** [review]

File: images/constants.yml [hunk 2]
```diff
@@ -43,32 +43,29 @@ variables:
...
+  NET_CORE_APP_60: 6.0.33
+  NET_CORE_APP_60_SHA: 0892015544d8903999f8e0fadab6b4b91eb180e495fa5e36c1a755b1d42e134858b7bdbfd60d1880650d9c528d07e31b9ccfc73e650e5d890a955902a89139cf
+  NET_CORE_APP_80: 8.0.8
+  NET_CORE_APP_80_SHA: 8f5220098c562fa3490417748eb9f4f9ca1551f7155728b9ebb1924359c63c18dedef643bcd89ec67b59cb5b1b9de7283ee156ef381ffb16801b516dba9b1b0f
+  NET_CORE_APP_90: 9.0.0-rc.1.24431.7
+  NET_CORE_APP_90_SHA: 9f9a85b8d9f6362ed2c2d0edefd04999181b2c386647644fbc1d9f248255387324399edb1c40bc7fa8c47adc22e2d71db5f25ce794521d59e46c40593b5f6cc5
   node18Version: 18.20.4
-  node20Version: 20.15.1
+  node20Version: 20.17.0
+  node22Version: 22.8.0
```
Comment (sarsharma) [inline images/constants.yml:60]: should we remove node22 till it reaches LTS status?

File: monitor_version_scripts/monitor_variables.sh
Comment (sarsharma) [inline monitor_version_scripts/monitor_variables.sh:1]: nit: just curious, will this empty line be added with each automated PR?
If so, can we fix it?

File: platforms/dotnet/versions/buster/versionsToBuild.txt [hunk 3]
```diff
@@ -61,6 +61,7 @@
 6.0.421, aa2c1fdc06c477acbb8ca938895f22373a96e96bb75028b496ddf3d433a1e347f3f765b414e8e09fbf1da1cc50f575e28572e701693bad2a33f9b92437a7d3fc,
 6.0.423, 4b4a0e66634211ae04fa030e18ae9e22640f5828307ba85c4bae596ab2d31260519197828dae3b2ec73d6772635e0b375536ea6591b03c67c2b7a5566f016952,
 6.0.424, e9823aa2ad261199f8289fde8721931c1e4d47357b4973b8c7d34c12abd440bb932064ac151b0e0d7b3d5b72a5dfe3f20d5dafa19e6f56f1a61ad54b7de5e584,
+6.0.425, a04b75af7c5850238a8d99a6f60b37753467db615831bb3833c14aec86faa2d6ee9b8643885798924a01e28acff44ac9ed39c89f7cbe53c5cb8753c802e85039,
```
Comment (sarsharma) [inline platforms/dotnet/versions/buster/versionsToBuild.txt:64]: not a blocker but let's follow up with another PR to stop building buster

File: images/constants.yml (hunk 2 above)
Comment (Saipriya-1144) [reply images/constants.yml:60]: This is just getting added in the constants file. we are not building node 22 image

**Review by sarsharma — APPROVED** [review]

**Review by CodingIsBliss — APPROVED** [review]

**Pull request merged**